i18n/locales/en/translations.ts

```diff
@@ -5,15 +5,15 @@ export const translations = {
     chapter: 'Chapter',
     challenge: 'Challenge',
     coming_soon: 'Coming soon',
-    start_chapter: 'Start chapter',
+    start_chapter: 'Start chapter'
   },
 
   chapter_one: {
     title: 'Secrets in plain sight',
     paragraph_one:
-      'The year is 2139. The very last bitcoin is slated to be mined in two weeks. For months, a final countdown has been taking place in Times Square for the last block when there will be officially no more bitcoin issued.',
-    paragraph_two: `And then suddenly, the network comes to a crawl. Instead of the normal ten-minute block interval, blocks are coming in just a few times a day, if not longer. Your mom's bitcoin node stops syncing. The media starts reporting on the situation. It's everywhere on social media.`,
-    paragraph_three: `The price of bitcoin is falling. There is a knock at your door, and when you open it, a yellowed, age-weathered envelope has been left for you. There's no name or return address. What could be inside?`,
+      'The year is 2139. The last bitcoin is two weeks from being mined. For months, a clock has ticked down in Satoshi Square. The world awaits the last block. Then, suddenly, the network grinds to a halt.',
+      paragraph_two: null,
```

Comment:
Same here:

<img width="626" alt="image" src="https://user-images.githubusercontent.com/10757768/223984344-a016f29c-ba07-465a-a736-a288d3bce3df.png">

Reply:
Resolved this in this PR by adding strings with just a space character for the two translation keys (so it renders two `p` tags with just a space in each). We can find a solution to make this template more flexible later.